version: '1'
name: data-analysis
services:
  jupyter:
     image: 'jupyterhub:my-jupyterhub'
     privileged: true
     volumes:
       - ./etc/jupyter/jupyterhub_config.py:/srv/jupyterhub/jupyterhub_config.py:ro
       - ./data/jupyterhub:/root/data:rw
     networks:
       - dev
     restart: always
     environment:
       - TZ=Asia/Shanghai
#       - GITLAB_HOST=https://gitlab.com
#       - OAUTH_CALLBACK_URL=https://www.yourdomain.com/hub/oauth_callback
#       - OAUTH_CLIENT_ID=b8312c7666acc33cf1263866cbd9bd72
#       - OAUTH_CLIENT_SECRET=1de38bad6cbd021c76afecbf7e336941
     ports:
       - "8000:8000"
       
       
networks:
  dev:
    external: true
